### PR TITLE
doc(stores): document how to handle the Mozilla review process

### PR DESCRIPTION
**Pull request by JalilArfaoui**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -222,3 +222,39 @@ extensions.lmem@gmail.com
 
 To generate your own tokens and deploy from your local environment :
 https://github.com/DrewML/chrome-webstore-upload/blob/master/How%20to%20generate%20Google%20API%20keys.md
+
+### Firefox Addons
+
+[Developer Hub](https://addons.mozilla.org/en-US/developers/addons)
+| account: infrastructure@lmem.net
+
+In order for the Mozilla review to complete successfully, please the following steps :
```
Comment (lutangar) [inline docs/CONTRIBUTING.md:231]: please the following steps -> please follow these steps
> or smthing like that

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -222,3 +222,39 @@ extensions.lmem@gmail.com
...
+
+- Use [SemaphoreCI](https://semaphoreci.com/lmem/extension/) to deploy to `Firefox production`
+- [Download](https://github.com/dis-moi/extension/releases) the source code archive of the version you want to deploy to production extension (`Source code (tar.gz)`)
+- Create a `buildDisMoi.sh` file on your machine with the following content :
```
Comment (lutangar) [inline docs/CONTRIBUTING.md:235]: Could we ease the process and prepare at least part of this script ?
Maybe embed a part of it in the `package.json` or a dedicated script in `bin` dir.

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -222,3 +222,39 @@ extensions.lmem@gmail.com
...
+Requirement: Docker
+
+Instructions:
+ - Extract archive
+ - Run ./buildDisMoi.sh
+```
+
+- Save changes
+
+Note: We plan to remove any SendInBlue direct call from extension soon.
```
Comment (lutangar) [inline docs/CONTRIBUTING.md:260]: Good to know :D

Could you format it as a note please? 
> `>`

**Review by lutangar — CHANGES_REQUESTED** [review]

**Comment by lutangar** [comment]
> This PR is the response to issue #743

**Comment by lutangar** [comment]
ping @JalilArfaoui

**Comment by lutangar** [comment]
@JalilArfaoui I'm going to take this back since *sendinblue* is going to be removed.
> see #161

However if we want to fully automate the build on Firefox there are still these issues which aren't resolved at __mozilla__:
This https://github.com/mozilla/addons-server/issues/9913 is blocking https://github.com/mozilla/sign-addon/issues/409 and thus blocking us.

**Comment by lutangar** [comment]
@JalilArfaoui be re-reading your doc, I wonder if we can remove the docker requirement, I think I remember an issue with node version...

**Comment by lutangar** [comment]
Also when trying to login as `infrastructure@lmem.net` to the **Firefox account** as mentionned in the doc I get the following error:
 ```
 Primary account email required for sign-in
 ```
 @MaartenLMEM @JalilArfaoui

**Comment by lutangar** [comment]
I'm closing this in favor of #800

**Comment by JalilArfaoui** [comment]
@lutangar It’s cool that we can get rid of the ugly `.sh` part … but I don’t get why this doc PR is moved into a backend PR about posting contributions ?

Let’s update this one instead, no ?

**Comment by lutangar** [comment]
@JalilArfaoui sorry wrong issue

> see  #161

**Comment by christpet** [comment]
@lutangar is this really in staging?

**Comment by lutangar** [comment]
Superseded by #800

**Pull request closed without merging**